python/ray/data/tests/test_metadata_provider.py

```diff
@@ -220,6 +220,12 @@ def test_fast_file_metadata_provider(caplog, fs, data_path, endpoint_url):
     assert meta.schema is None
 
 
+def test_fast_file_metadata_provider_ignore_missing():
+    meta_provider = FastFileMetadataProvider()
+    with pytest.raises(ValueError):
+        meta_provider.expand_paths([], None, ignore_missing_paths=True)
```

Comment:
Could you add a unit test that path expansion passes with the `DefaultFileMetadatProvider()` and `ignore_missing_paths=True`, and that the expected (existing) paths are returned?

Reply:
added